app/helpers/application_helper/button/vm_snapshot_revert.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::VmSnapshotRevert < ApplicationHelper::Button::Basic
```

Comment:
@h-kataria Ah, I see!  I'm a bit nervous about making that change, because that's more about updating the other provider's VM code to use the supports mixin, taking out their validate methods, etc.